storm/src/main/storm/mesos/MesosNimbus.java

```diff
@@ -232,21 +265,22 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
       if (_offers == null) {
         return;
       }
-      LOG.debug("resourceOffers: Currently have " + _offers.size() + " offers buffered" +
-          (_offers.size() > 0 ? (":" + offerMapToString(_offers)) : ""));
+      LOG.debug("resourceOffers: Currently have {} offers buffered {}",
+                _offers.size(), (_offers.size() > 0 ? (":" + offerMapToString(_offers)) : ""));
+
       for (Protos.Offer offer : offers) {
         if (isHostAccepted(offer.getHostname())) {
-          LOG.debug("resourceOffers: Recording offer from host: " +
-              offer.getHostname() + ", offerId: " + offer.getId().getValue());
+          LOG.debug("resourceOffers: Recording offer from host: {}, offerId: {}",
+                    offer.getHostname(), offer.getId().getValue());
           _offers.put(offer.getId(), offer);
         } else {
-          LOG.debug("resourceOffers: Declining offer from host: " +
-              offer.getHostname() + ", offerId: " + offer.getId().getValue());
+          LOG.debug("resourceOffers: Declining offer from host: {} offerId: {}",
```

Comment:
minor: `,` has been removed, and is inconsistent with previous debug message for "Recording offer".  i.e., either remove comma from both, or leave in both.